packages/aws-rfdk/lib/lambdas/nodejs/lib/deadline-client/deadline-client.ts

```diff
@@ -150,38 +179,51 @@ export class DeadlineClient {
     return options;
   }
 
+  private async performRequestWithRetry(options: https.RequestOptions, retriesLeft: number, retryDelayMs: number, data?: string): Promise<Response> {
+    return this.performRequest(options, data)
```

Comment:
Just curious... why this style of syntax instead of something like:
```ts
try {
  return await this.performRequest(options, data);
} catch (...) {
 ... exception handling
}
```

Reply:
Just because of the historical reasons. I like your style more, so will use it instead.